web/components/admin/ApplicantInfo/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { firebaseConnect, firestoreConnect } from 'react-redux-firebase';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import applicantCollections from '../../../util/applicantCollections';
+
+const mapPropsToQueries = (props) => {
+  const { applicantType, applicantId } = props;
+  return [
+    { collection: applicantCollections[applicantType].shortInfo, doc: applicantId },
+    { collection: applicantCollections[applicantType].longInfo, doc: applicantId },
+  ];
+};
+
+class ApplicantInfo extends React.Component {
+  getApplicantShortInfo = () => {
+    const { firestore, applicantId, applicantType } = this.props;
+    if (!applicantId || !applicantType) {
+      return null;
+    }
+    const shortInfo = firestore.ordered[applicantCollections[applicantType].shortInfo];
```

Comment:
Why are we using `ordered` here? I think we should use `data` here since we're retrieving a single document. `ordered` is only useful for retrieving a list in order, and takes `O(n)` time to get a document with the right id

Reply:
Oops, fixed